.github/workflows/test.yml

```diff
@@ -31,11 +31,11 @@ jobs:
             name: system - support users
             params: 'spec/system/support_users'
           -
-            name: system - other & a11y
-            params: '--exclude-pattern "spec/system/{jobseekers,publishers,support_users}, spec/system/{jobseekers,publishers}_*_spec.rb" spec/system spec/accessibility'
+            name: system - other
+            params: '--exclude-pattern "spec/system/{jobseekers,publishers,support_users}, spec/system/{jobseekers,publishers}_*_spec.rb" spec/system'
```

Comment:
I'm not entirely sure how this works, but I see 'exclude pattern', so I imagine that any test with a path matching the pattern given is not run.

Does this mean `unit` and `system - other` are both running our unit tests?

Reply:
not entirely sure, i sort of search and replaced. ill see on this run of PR actions...